### PR TITLE
Create ib-tauris-note.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/67382/style-request-ib-tauris

In their guidelines they also talk about a Harvard style, hence this is labelled Note.

Guidelines: [IBT House Style 2016.pdf](https://github.com/citation-style-language/styles/files/1355290/IBT.House.Style.2016.pdf)

### CODE REVIEW

**Comment by adam3smith** [comment]
two quick comments; looks great otherwise.

**Comment by rmzelle** [comment]
Uhmm, @adam3smith, where are those two comments?

**Comment by adam3smith** [comment]
sorry about that; posted now.

File: ib-tauris-note.csl [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    <group delimiter="/">
+      <number variable="volume" form="roman"/>
+      <text variable="issue"/>
+    </group>
+  </macro>
+  <macro name="container">
+    <text variable="container-title" font-style="italic"/>
+  </macro>
+  <macro name="editors">
+    <names variable="editor translator">
```
[the remark this answers is not in the document]
Comment (POBrien333) [reply ib-tauris-note.csl]: Sorry. Is there a word missing? Not getting it.

File: ib-tauris-note.csl [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      <choose>
+        <if position="ibid-with-locator">
+          <group delimiter=", ">
+            <text term="ibid"/>
+            <group delimiter=" ">
+              <label variable="page" form="short"/>
+              <text variable="locator"/>
+            </group>
+          </group>
+        </if>
```
[the remark this answers is not in the document]
Comment (POBrien333) [reply ib-tauris-note.csl]: thanks. added.

File: ib-tauris-note.csl (hunk 1 above)
[the remark this answers is not in the document]
Comment (adam3smith) [reply ib-tauris-note.csl]: sorry, forgot the code tags

**Comment by adam3smith** [comment]
(the cs:names/cs:name construction isn't the most fortunate design decision in CSL -- see my just committed fix for how this should look)

**Comment by POBrien333** [comment]
I should've maybe tested that before just adding random code. haha. my fault.

**Comment by adam3smith** [comment]
@rmzelle good to merge from my end

**Comment by rmzelle** [comment]
Here too. Just waiting on the forums to see if we use a better documentation link.

**Comment by POBrien333** [comment]
This is ready now from my side :)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**